packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -49,14 +39,9 @@ describe('ReactDOMInput', () => {
     var node = ReactDOM.findDOMNode(stub);
     expectDev(console.error.calls.count()).toBe(1);
 
-    // Simulate a native change event
-    setUntrackedValue(node, 'giraffe');
-
     // This must use the native event dispatching. If we simulate, we will
     // bypass the lazy event attachment system so we won't actually test this.
-    var nativeEvent = document.createEvent('Event');
-    nativeEvent.initEvent('change', true, true);
-    node.dispatchEvent(nativeEvent);
+    ReactTestUtils.SimulateNative.change(node, {target: {value: 'giraffe'}});
```

Comment:
How do we know that this actually does anything at all? I'm not sure this actually tests the behavior we're trying to test. Since this is a no-change test.

Reply:
Correct me if I'm wrong, but `ReactTestUtils.SimulateNative.change` actually needs to have an element listening to an event to be applied. Any __change__ dispatched to this element won't have any consequence (_the value will still be the same_). So, judging that in this case, where there's no event at all, makes more sense this way.

Reply:
@sebmarkbage Thank you for the hint with `setUntrackedValue.call(node, 'giraffe');`!
It's way more flexible too! Working on right now!